templates/email/default/waste/other-reported.html

```diff
@@ -16,10 +16,20 @@ <h1 style="[% h1_style %]">Your request has been&nbsp;logged</h1>
   <h1 style="[% h1_style %]">Your report has been&nbsp;logged</h1>
  [% END %]
 
-[% IF cobrand.moniker == 'sutton' %]
-<p style="[% p_style %]">Thank you for contacting the London Borough of Sutton.</p>
+[% IF report.category == 'Request new container' %]
```

Comment:
This is adding to the `<h1>` already present above, rather than changing it (rebase issue I assume).